tests/installation/disable_grub_timeout.pm

```diff
@@ -21,15 +21,6 @@ use version_utils qw(is_sle is_leap);
 sub run {
     my ($self) = shift;
 
-    if (check_var('ARCH', 's390x')) {
```

Comment:
We don't cover zVM here, let's either enable it for svirt backend only, or address zVM too. 

@Soulofdestiny WDYT?

Reply:
zVM was already covered. The task was to implement it for kvm also.